housing_cooperative_base/models/cellar.py

```diff
@@ -10,9 +10,8 @@
 class Cellar(models.Model):
     _name = "hc.cellar"
     _description = "Cellar"
+    _inherit = "hc.premise"
```

Comment:
I re-read [the official documentation](https://www.odoo.com/documentation/12.0/reference/orm.html#reference-orm-inheritance) and I think your right. We shouldn't user `_inherits` as I previously said.

:+1::+1: